### DISCUSSION TITLE
Is it possible to add a secondary replication target?

### DISCUSSION THREAD

**Discussion by bernardx** [minio/minio#11956] [Q&A] [answered]
I would like to replicate my local bucket to two different remote offices.

I was able to set up the first one with this command: 
`mc admin bucket remote add minio/test http://remote-1/test --service "replication" `

However, while adding the second one:
`mc admin bucket remote add minio/test http://remote-2/test --service "replication" `
I got:  **The remote target with this label already exists.**

Does Minio support multiple replication targets?

**Comment by harshavardhana** [comment, marked as answer]
No it is not possible @bernardx
